### DISCUSSION TITLE
First created user seems not to be an admin

### DISCUSSION THREAD

**Discussion by 5chmidti** [mealie-recipes/mealie#2392] [Q&A] [answered]
After initial install, I wanted to check the [admin settings](https://nightly.mealie.io/documentation/getting-started/features/#server-administration), but couldn't find any direct link inside the UI. I tried using the url path '/admin/site-settings', but that doesn't work (shows the UI without any valid data) because of `mealie-api         | INFO:     172.21.0.4:41422 - "GET /api/admin/groups?page=1&perPage=-1 HTTP/1.1" 403 Forbidden`. That's why I think the first created user is not an admin account.

Is there a default admin account, or am I missing something?

Tried both omni-nightly and the split nightly builds.

**Comment by hay-kot** [comment, marked as answer]
There is a default admin account

https://nightly.mealie.io/documentation/getting-started/installation/installation-checklist/#step-3-startup

**Reply by 5chmidti** [reply]
My bad, thanks
